src/SFA.DAS.ApprenticeCommitments/Application/Commands/CreateApprenticeship/CreateApprenticeshipCommandHandler.cs

```diff
@@ -5,44 +5,45 @@
 using System.Threading.Tasks;
 using SFA.DAS.ApprenticeCommitments.Application.Services.ApprenticeLogin;
 using SFA.DAS.ApprenticeCommitments.Apis.InnerApi;
+using SFA.DAS.ApprenticeCommitments.Apis.TrainingProviderApi;
 
 namespace SFA.DAS.ApprenticeCommitments.Application.Commands.CreateApprenticeship
 {
     public class CreateApprenticeshipCommandHandler : IRequestHandler<CreateApprenticeshipCommand>
     {
         private readonly ApprenticeCommitmentsService _apprenticeCommitmentsService;
         private readonly CommitmentsV2Service _commitmentsService;
+        private readonly TrainingProviderService _trainingProviderService;
         private readonly ApprenticeLoginService _apprenticeLoginService;
 
         public CreateApprenticeshipCommandHandler(
             ApprenticeCommitmentsService apprenticeCommitmentsService,
             ApprenticeLoginService apprenticeLoginService,
-            CommitmentsV2Service commitmentsV2Service)
+            CommitmentsV2Service commitmentsV2Service,
+            TrainingProviderService trainingProviderService)
         {
             _apprenticeCommitmentsService = apprenticeCommitmentsService;
             _apprenticeLoginService = apprenticeLoginService;
             _commitmentsService = commitmentsV2Service;
+            _trainingProviderService = trainingProviderService;
         }
 
         public async Task<Unit> Handle(
             CreateApprenticeshipCommand command,
             CancellationToken cancellationToken)
         {
+            (TrainingProviderResponse trainingProvider, Apis.CommitmentsV2InnerApi.ApprenticeshipResponse apprentice) = await GetExternalData(command);
```

Comment:
It's possible to use `var` here too:

```suggestion
            var (trainingProvider, apprentice) = await GetExternalData(command);
```

Reply:
thx, great suggestion